data/i18n/ar/ar.toml

```diff
@@ -0,0 +1,555 @@
+other = "جاري جمع النتائج..."
+
+[legacy_repos_message]
+other = """تم إهمال مستودعات الحزم القديمة (`apt.kubernetes.io` و `yum.kubernetes.io`) عن الخدمة ابتداءً من ١٣ سبتمبر ٢٠٢٣، فيوصى بشدةٍ استخدام المستودع الجديد (`pkgs.k8s.io`) حيث أنه من الوارد تعطيل المستودعات القديمة دون سابق إنذار. يوفر المستودع الجديد كل إصدارات كوبرنيتيز ابتداءً من `v1.24.0`. """
+
+[main_by]
+other = "by"
+
+[main_cncf_project]
+other = """نحن <a href="https://cncf.io/">CNCF</a> مشروع متخرج</p>"""
+
+[main_community_explore]
+other = "تعرف على المجتمع"
+
+[main_contribute]
+other = "ساهم"
+
+[main_copyright_notice]
+other = """مؤسسة لينَكس &reg;. جميع الحقوق محفوظة. لدى مؤسسة لينكس علامات تجارية مسجلة وتستخدم العلامات التجارية .للحصول على قائمة العلامات التجارية لمؤسسة لينكس يرجى الاطلاع على <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">صفحة استخدام العلامات التجارية</a>"""
+
+[main_documentation_license]
+other = """مطورو كوبرناتيز | تم توزيع الوثائق تحت رخصة <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC ب 4.0</a>"""
+
+[main_github_invite]
+other = "هل أنت مهتم بتطوير مشروع كوبرنيتيز؟"
+
+[main_github_view_on]
+other = "(GitHub) اعرضه على جيت هاب"
+
+[main_kubernetes_features]
+other = "مميزات كوبرنيتيز"
+
+[main_kubeweekly_baseline]
+other = "هل أنت مهتم بتلقي آخر أخبار كوبرنيتيز؟ سجل اشتراكك بنشرة KubeWeekly الأسبوعية."
+
+[main_kubernetes_past_link]
+other = "تصفح إصدارات النشرة"
+
+[main_kubeweekly_signup]
+other = "سجل اشتراكك"
+
+[main_page_history]
+other = "تاريخ الصفحة"
+
+[main_page_last_modified_on]
+other = "آخر تحديث في"
+
+[main_read_about]
+other = "اقرأ عن"
+
+[main_read_more]
+other = "اقرأ المزيد"
+
+[not_applicable]
+# Localization teams: it's OK to use a longer text here
+other = "لا ينطبق"
+
+[note]
+other = "ملاحظة:"
+
+[objectives_heading]
+other = "الأهداف"
+
+[options_heading]
+other = "الإعدادات"
+
+[outdated_blog__message]
+other = "يعود نشر هذا المقال إلى أكثر من عام، فقد يحتوي على معلومات قديمة. تأكد من موافقة معلومات هذا المقال أحدث الوثائق المتوفرة لضمان صحتها."
+
+[patch_release]
+other = "الإصدار التصحيحي"
+
+[post_create_issue]
+other = "أنشئ قضية (issue)"
+
+[prerequisites_heading]
+other = "قبل أن تبدأ"
+
+[previous_patches]
+other = "إصدارات التصحيح (patch):"
+
+# The following text is displayed when JavaScript isn't available.
+[release_binary_alternate_links]
+other = """بإمكانك تحميل ملفات تشغيل كوبرنيتيز ورموز التحقق (checksums) الخاصة بها من خلال الروابط الموجودة في [سجل التغيرات (CHANGELOG)](https://github.com/kubernetes/kubernetes/tree/master/CHANGELOG).
+بإمكانك أيضاً زيارة [downloadkubernetes.com](https://www.downloadkubernetes.com/) للبحث عن الملفات الخاصة بإصدار معين أو ببنية حاسوبية معينة."""
+
+[release_binary_arch]
+other = "بنية الحاسوب"
+
+[release_binary_arch_option]
+other = "بنيات الحاسوب"
+
+[release_binary_copy_link]
+other = "انسخ الرابط"
+
+[release_binary_copy_link_certifcate]
+other = "cert"
+
+[release_binary_copy_link_checksum]
+other = "checksum"
```

Comment:
```suggestion
```

When we don't have a string chosen yet, we can omit it, or put in a placeholder comment.